<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MQTT测试</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连接状态 -&gt; 连接失败！！！</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>订阅Topic：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_topicSub">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>QoS：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_qosSub">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>订阅</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_topicPub">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_qosPub">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>QoS：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发布Topic：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发布</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内容：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
